_posts/en/newsletters/2022-03-30-newsletter.md

```diff
@@ -0,0 +1,151 @@
+  discussing how to revise the LN gossip protocol used to advertise
+  information about available payment channels.  In particular, this
+  week saw two active threads:
+
+    - *Major update:* in [response][osuntokun gossip2] to Rusty
+      Russell's [major update][russell gossip2] proposal from last month,
+      Olaoluwa Osuntokun repeatedly expressed concern with an aspect of
+      the proposal that would introduce plausible deniability in the
+      link between onchain funds and a specific LN channel.  That
+      capability would also make it easier for non-LN users to advertise
+      the existence of channels that might not actually exist, which
+      could degrade the ability for a spender to find a working path
+      across the network to the node receiving the funds.
+
+    - *Minor update:* Osuntokun [posted][osuntokun gossip1.1] a separate
+      proposal for a much smaller update to the gossip protocol aimed
+      mainly at allowing taproot-based channels.  The proposal uses
+      [MuSig2][topic musig] to allow a single signature to prove
+      authorization related to all four public keys involved (two node
+      identifier keys, two channel-spending keys) and would likely
+      require that the channel setup transaction be spendable using
+      MuSig2.
+
+        He also suggested that it might be useful to add an SPV partial
+        merkle branch proof to the channel announcement message.  This
+        would prove that the channel setup transaction was included in a
+        block, eliminating lightweight clients from having to download
+        the entire block containing the transaction in order to verify
+        its existence.
+
+## Selected Q&A from Bitcoin Stack Exchange
+
+*[Bitcoin Stack Exchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+<!-- FIXME: harding to update Tuesday -->
+
+- [BDK 0.17.0][] is a release of this library for building Bitcoin
+  wallets.  Improvements in this version should make it easier
+  to derive addresses even when the wallet is offline.
+
+- [Bitcoin Core 23.0 RC1][] is a release candidate for the next major
+  version of this predominate full node software.  The [draft release
+  notes][bcc23 rn] list multiple improvements that advanced users and
+  system administrators are encouraged to test before the final release.
```

Comment:
Is this a section that the [Review Club meeting for testing the release candidate](https://bitcoincore.reviews/v23-rc-testing) could be mentioned? Maybe as an extra guidance for people that want to help with testing.

Reply:
The guide is now also on the dev wiki